### ISSUE TITLE
Does not build on arch != amd64

### ISSUE THREAD

**Issue by williamdes** [progval/unicode_names2#40]
See: https://github.com/sudo-bot/docker-rustpython/actions/runs/9814318960/job/27101759812

```
error[E0412]: cannot find type `u512` in this scope
#15 559.6      --> /workdir/target/release/build/unicode_names2-851dc8f074cc9f46/out/generated.rs:13939:43
#15 559.6       |
#15 559.6 13939 | pub static PHRASEBOOK_OFFSETS2: &'static [u512] = &[
#15 559.6       |                                           ^^^^ not found in this scope
#15 559.6 
```

**Comment by progval** [comment]
This is fixed in v1.2.0 (thanks to #34), but you are trying to build RustPython, which has v1.1.0 in its lockfile.

**Comment by williamdes** [comment]
Thank you for the quick reply !
